notebooks/official/training/xgboost_data_parallel_training_on_cpu_using_dask.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    IRIS_DATA_FILENAME = 'gs://cloud-samples-data/ai-platform/iris/iris_data.csv'
</pre><p>Prefer using penguins dataset as there is existing controversy with using iris. Not a hard blocker though.</p><p> </p><p>gs://cloud-samples-data/ai-platform/penguins/</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>I'll use IRIS data here. Since other notebook official/migration/UJ9 Vertex SDK Custom XGBoost with pre-built training container.ipynb is using iris with xgboost. So we may want to keep them sync. </p>